docs/build-from-scratch.md

```diff
@@ -16,6 +16,8 @@
 
 ## Getting the source
 
+### Latest (Recommended)
```

Comment:
To make it super clear we are talking about two different options, I would make this line something like `Option 1) get latest (Recommended). Then on the next item `Option 2) Release Archive

Reply:
Please note: I made only the first letter capitalized above.

Reply:
Can do.